pkg/monitoring/client/prometheus/prometheus.go

```diff
@@ -54,9 +59,35 @@ var (
 		},
 		[]string{"code", "method", "host"},
 	)
+
+	resourceRequestsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "rest_client_operation_and_resource_requests_total",
+		Help: "Number of HTTP requests partitioned by the requested resource name and operation.",
```

Comment:
>Number of HTTP requests partitioned by the requested resource name and operation
>
Should this be characterized differently since we're exposing this as "operation"?

Reply:
i renamed the metric a bit and transitioned to the term `verb`. What did you have in mind here?

Reply:
As in, should `Number of HTTP requests` be `Number of operators` or `verbs`?

Reply:
that would be more accurate, since this metric isn't even guaranteed to capture all http requests in the event that the request isn't for a k8s resource

Reply:
i changed the metric name to `kubevirt_rest_client_resource_requests_total` and reflected in the help text that this metric counts the number of resource related requests